Dockerfile.bundle

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=numaresources-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.36.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.25.0
```

Comment:
minor: why going back?

Reply:
this was generated automatically by running `make generate bundle manifests`. I'll check and fix it, thanks for raising.